examples/features/effect-handlers.effekt.md

```diff
@@ -93,3 +97,65 @@ def genFibs(limit: Int): List[Int] / {} = {
 ```effekt:repl
 genFibs(15)
 ```
+
+## Singleton operation
+
+Often you want to declare an interface that is entirely defined by just one operation. In this case you can declare it as a singleton operation:
+
+```effekt:hide
+extern def time(): Int =
+  js "Date.now()"
+```
+
+```
+effect tell(): Int
+```
+
+The handler for this interface uses a slightly different more concise syntax like it is used in the literature:
+
+```
+def tellTime[A] { prog: () => A / tell }: A =
+  try { prog() }
+  with tell { () =>
+    resume(time())
+  }
+```
+
+```effekt:repl
+tellTime {
+  val start = do tell()
+  val end = do tell()
+  println("took " ++ show(end - start))
+}
+```
+
+## `with` handler
+
```

Comment:
Missing text?
``````suggestion
## `with` handler

If you just want to use one handler for the rest of the given scope, and/or you want to reduce nesting, use `with` syntax where:
```effekt:sketch
with limited[Int](1);
...
```
is the same as
```effekt:sketch
limited[Int](1) {
  ...
}
```
``````
(The text could probably be better, this was the first quick thing I wrote down).

Reply:
Maybe we could even reuse the `tellTime` example above and just have the exact same code, but now with `with`:
```
with tellTime;
val start = do tell()
val end = do tell()
println("took " ++ show(end - start))
```

That would probably make it easier and we wouldn't need to introduce a new -- albeit nice! -- handler.